packages/wrangler/src/api/integrations/platform/index.ts

```diff
@@ -16,10 +16,17 @@ import { getLegacyAssetPaths, getSiteAssetPaths } from "../../../sites";
 import { CacheStorage } from "./caches";
 import { ExecutionContext } from "./executionContext";
 import { getServiceBindings } from "./services";
-import type { Config } from "../../../config";
+import type { Config, RawConfig, RawEnvironment } from "../../../config";
 import type { IncomingRequestCfProperties } from "@cloudflare/workers-types/experimental";
 import type { MiniflareOptions, ModuleRule, WorkerOptions } from "miniflare";
 
+export { readConfig as unstable_readConfig };
+export type {
+	Config as UnstableConfig,
```

Comment:
Quick question, should we use `Unstable_Config`, etc. similar to `unstable_readConfig`?

Reply:
In the future there might actually be a type that starts with Unstable...

Reply:
I was going to do that but noticed there are already some types named like this without the underscore e.g. `UnstableDevOptions`, `UnstableDevWorker`. Thought it was better to keep it consistent.

Reply:
Oh... 😢 
I think that was probably a bad idea. But I guess it's best to stick with the convention.

Reply:
You can change them all if you like (they're called unstable so people can't exactly say they haven't been warned). That should probably be in a separate PR though.